fri/src/two_adic_pcs.rs

```diff
@@ -329,19 +334,27 @@ where
             .map(|(query_opening, &index)| {
                 let mut ro = [Challenge::zero(); 32];
                 let mut alpha_pow = [Challenge::one(); 32];
+
                 for (batch_opening, (batch_commit, mats)) in izip!(query_opening, &rounds) {
-                    let batch_dims: Vec<Dimensions> = mats
+                    let batch_heights = mats
                         .iter()
-                        .map(|(domain, _)| Dimensions {
-                            // todo: mmcs doesn't really need width
-                            width: 0,
-                            height: domain.size(),
```

Comment:
I think this was a bug - we want to pass the MMCS the codeword length, but `domain.size()` is the dimension. It wasn't causing errors before since `FieldMerkleTreeMmcs` only uses heights for relative ordering.